extensions/default/src/getHangingProtocolModule.js

```diff
@@ -92,211 +94,6 @@ const defaultProtocol = {
       ],
       createdDate: '2021-02-23T18:32:42.850Z',
     },
-
```

Comment:
This really doesn't belong in the default HP, and the @ohif/mnGrid hp is now available on the URL, so can be directly applied.